config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -1220,3 +1920,235 @@ fields:
+        type: chado_property_type_default
+        description: The genus of the germplasm that data was collected for (i.e. that the experiment was done on). If data was collected for multiple genuses then each genus should be its own entry.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: TAXRANK
+            termAccession: '0000005'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_germspecies
+        content_type: biochem_experiment
+        label: Germplasm Species
+        type: chado_property_type_default
+        description: The species of the germplasm that data was collected for (i.e. that the experiment was done on). If data was collected for multiple species, then each species should be added as its own entry.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: TAXRANK
+            termAccession: '0000006'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_germcollection
+        content_type: biochem_experiment
+        label: Germplasm Collection
+        type: chado_property_type_default
+        description: The official name of the collection of germplasm used in this experiment (e.g. LR-69; AGILE LDP).
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: local
+            termAccession: 'project_germcollection'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_featureofinterest
+        content_type: biochem_experiment
+        label: Key Measurements
+        type: chado_property_type_default
+        description: List the full names of the core measurements being taken in this experiment. If you are exploring protein concentration then examples can be "Alanine content (% on dry basis) in whole lentils".
```

Comment:
![Screen Shot 2024-07-08 at 12 44 46](https://github.com/TripalCultivate/TripalCultivate/assets/45402954/eb3fb7f5-c905-4021-b632-132501efcfec)
![Screen Shot 2024-07-08 at 12 48 53](https://github.com/TripalCultivate/TripalCultivate/assets/45402954/70b7c865-3689-43e4-a37f-883eb2360917)

Thank you for the suggestion! However, the .yml seem doesn't like colon, so I worked around it by deleted colon, hope that's close enough!

Reply:
Now it looks like`"Alanine content (% on dry basis) in whole lentils".` was cut off completely 😢  I'm not sure the colon is the problem since it was used above in the ratio of 2:1:1 example. But, if you can manage to get the example to show again I agree it wouldn't need the colon to make sense!

Reply:
The colon is only an issue because it's not in quotes. It should be 

> description: 'List the full names of the core measurements being taken in this experiment. For example, if you are exploring protein concentration in lentils, then a core measurement could be: "Alanine content (% on dry basis) in whole lentils".'